games/game_witcher3.py

```diff
@@ -11,20 +15,32 @@ class Witcher3Game(BasicGame):
 
     Name = "Witcher 3 Support Plugin"
     Author = "Holt59"
-    Version = "1.0.0a"
-    Description = "The Description Of The Dead"
+    Version = "1.0.1"
```

Comment:
Use semantic versioning. Leve the a for alpha (this isn't release version) and increase minor (second number) version since you added new functionality instead of just bugfixing.